main.C

```diff
@@ -539,38 +539,38 @@ LOCALFUN TAG *ReadAndProcessApeHeader(fstream &input) {
 
   input.seekg(-(INT32)(length + offset), ios::end);
 
-  char *const buffer = new char[length];
+  const char *items_buffer = new char[length];
```

Comment:
pre-existing, but should this buffer be freed before we return?

Reply:
I've changed the variable name to more clearly indicate that it's no longer a buffer.